apollo-router/src/apollo_studio_interop/mod.rs

```diff
@@ -523,1423 +523,3 @@ fn format_value(value: &Value, f: &mut fmt::Formatter) -> fmt::Result {
         rest => f.write_str(&rest.to_string()),
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Retain these two lines to include the `tests` module for testing.
e.g.:
```
#[cfg(test)]
mod tests;
```

Reply:
also, you need to rename `test.rs` to `tests.rs`.

Reply:
Thank you! I tried some of these but not at the same time and I also was always missing the #[cfg(test)]; mod tests;` part.